runeq/v2sdk/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""
+Version 2 client for the Rune analysis API.
+
+"""
+from sys import stderr
+
+from runeq import Config
+from runeq.v2sdk import client
+
+
+_beta_warned = False
+"""
+Whether the warning about beta usage has been printed.
+
+"""
+
+
+def beta_warning():
+    """
+    Issue a warning about using SDK code in beta testing.
+
+    """
+    global _beta_warned
+
+    if not _beta_warned:
+        print(
+            "WARNING: You are accessing a part of the Rune SDK that is "
+            "currently in Beta. Please be aware that Beta functionality is "
+            "subject to change in future releases, and may break your code "
+            "upon upgrade.",
+            file=stderr,
+        )
+        _beta_warned = True
+
+
+def initialize(*args, **kwargs):
```

Comment:
This is a pretty high-level question, but I don't think I see an answer in the design doc: Is there a reason that the SDK is based around global state and an `initialize` function, rather than being a bit more object oriented (where you instantiate a client and call methods on that)?

Reply:
@cranti suggested this idea in [this comment](https://docs.google.com/document/d/1D2WFmc4OgSssKe7Ae9FrvpoTItAS_k6245Yk2UZ_93M/edit?disco=AAAAcWo1cvk) on an older design doc.